### PR TITLE
Fix User Guide

### LINKED ISSUE #226: [PE-D][Tester F] Person that does not exist is mistaken as existing
There was an issue faced when adding a person that does not exist

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: testing1234567891011121314/ped#7

### LINKED ISSUE #237: [PE-D][Tester D] Some courses that has a flag in its name might not be allowed
Doesn't allow users to add courses that may share a flag. for example, fintech t/ might be a course

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: brendanng7/ped#10

### PR DESCRIPTION
**Pull request by hpcman**
Fixes #226. Fixes #237.

### CODE REVIEW

**Review by izruff — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -240,7 +241,7 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL c/COURSE [t/TAG]…​`
 Examples:
 
 - `add n/John Doe p/98765432 e/johnd@example.com c/CS2103/T`
-- `add n/Betsy Crowe t/friend e/betsycrowe@example.com c/CS1231S p/1234567 t/needs help`
+- `add n/Betsy Crowe t/friend e/betsycrowe@example.com c/CS1231S p/1234567 t/strruggling`
```
Comment (izruff) [inline docs/UserGuide.md:244]: I missed this one, pls fix

**Review by izruff — CHANGES_REQUESTED** [review]
Missed out one thing

File: docs/UserGuide.md [hunk 2]
```diff
@@ -576,6 +578,7 @@ By following these steps, you can easily transfer all your TAHub data to a new c
 
 1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
 2. **If you minimize the Help Window** and then run the `help` command (or use the `Help` menu, or the keyboard shortcut `F1`) again, the original Help Window will remain minimized, and no new Help Window will appear. The remedy is to manually restore the minimized Help Window.
+3. **If the name, tag, or course has known prefix**, the application will interpret it as parameters. The remedy is to avoid using any known prefixes as parameters.
```
Comment (izruff) [inline docs/UserGuide.md:581]: Oh can you also add an example to this? e.g. "c/COURSE e/SOMETHING" or sth

It's not clear to a new user what this means

**Review by izruff — CHANGES_REQUESTED** [review]

**Comment by izruff** [comment]
I think for the example, a full command is better. I'll edit it for ya

**Review by izruff — APPROVED** [review]
LGTM

**Review by juancarlovieri — APPROVED** [review]

**Pull request merged**